docs/static/include/pluginbody.asciidoc

```diff
@@ -852,7 +852,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "{plugintype}" }
 
   # Gem dependencies
-  s.add_runtime_dependency 'logstash-core', '>= 1.4.0', '< 2.0.0'
+  s.add_runtime_dependency 'logstash-core', '>= 5.0.0', '< 6.0.0'
```

Comment:
This should be replaced by line:
`s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"`